aspnetcore/blazor/fundamentals/routing.md

```diff
@@ -116,6 +116,18 @@ The router uses route parameters to populate the corresponding component paramet
 
 Optional parameters aren't supported. Two `@page` directives are applied in the previous example. The first permits navigation to the component without a parameter. The second `@page` directive takes the `{text}` route parameter and assigns the value to the `Text` property.
 
+## Reserved routing names
+
+The following keywords are reserved names and can't be used as route names or parameters:
+
+* `action`
+* `area`
+* `controller`
+* `handler`
+* `page`
+
+[!INCLUDE[](~/includes/reservedRouting.md)]
```

Comment:
This INCLUDE can't be part of the Blazor-focused doc in that form (i.e., covering the RP-MVC aspects). Draft this section with only Blazor-focused content.

Also, this doesn't seem right (for parameters at least) ...

```razor
@page "/TestReservedParam"
@page "/TestReservedParam/{action}"

<h3>Test Param</h3>

<p>
    Action: @Action
</p>

@code {
    [Parameter]
    public string Action { get; set; }

    protected override void OnInitialized()
    {
        Action = Action ?? "Not present";
    }
}
```

<img width="459" alt="Capture" src="https://user-images.githubusercontent.com/1622880/94846125-3e72bb80-03e6-11eb-8cbc-8117699e05b1.PNG">

I don't recall messing around much (at least not in a long time) with named routes in Blazor with either hosting model. I'd need to dig a little deeper on that before I can remark on it.